### PR TITLE
Add 'name' process tamplate tag

### PR DESCRIPTION
**Pull request by jkokosar**

### CODE REVIEW

**Comment by mstajdohar** [comment]
Write tests.

File: resolwe/flow/templatetags/process_fields.py [hunk 1]
```diff
@@ -6,10 +6,31 @@
...
 
 
+def get_data_attr(data, attr):
+    if isinstance(data, dict):
+        # `Data` object's id is hydrated as `__id` in expression engine
+        data = data['__id']
+
+    data_obj = Data.objects.filter(id=data)
+
+    if not data_obj.exists():
```
Comment (mstajdohar) [inline resolwe/flow/templatetags/process_fields.py]: We call `exists` and `first`. Are we doing 2 queries to DB or does Django handle this efficiently? Wouldn't it be more efficient to just call `first` with a try/catch?

File: resolwe/flow/templatetags/process_fields.py [hunk 2]
```diff
@@ -6,10 +6,31 @@
 """
 from django import template
 
+from resolwe.flow.models import Data
+
 
 register = template.Library()  # pylint: disable=invalid-name
 
 
+def get_data_attr(data, attr):
```
Comment (mstajdohar) [inline resolwe/flow/templatetags/process_fields.py]: Start private functions with an underscore.

File: resolwe/flow/tests/test_templatetags.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from resolwe.flow.managers import manager
+from resolwe.flow.models import Process, Data
+
+
+class ProcessFieldsTagsTest(TestCase):
```
Comment (mstajdohar) [inline resolwe/flow/tests/test_templatetags.py]: A long test for such small functionality. Couldn't you just create a Data obj and call Django's render function? What about border cases, e.g. no data object?

Comment (dblenkus) [inline resolwe/flow/tests/test_templatetags.py]: Problem is that `Data` objects are hydrated in a special way before rendering (i.e. `id` becomes `__id`). I can fake that, but I've decided to run real process as this is one of the significant things that are being tested.

File: resolwe/flow/templatetags/process_fields.py (hunk 1 above)
Comment (dblenkus) [inline resolwe/flow/templatetags/process_fields.py]: +1

**Pull request merged**